src/genie/libs/parser/iosxr/tests/ShowRibTables/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,59 @@
+expected_output = {
+
+        'rib_tables': {
```

Comment:
schema looks good. just FYI. technically we don't need `rib_tables` key and can start like below. but up to you if you fix. no harm with additional key. 
```
{
    'table_id': {

Reply:
Thanks Takashi, I am happy to make it the best it can be. I've changed it to remove "rib_tables" and it now starts with "table_id"